src/core/exposeEnablements.ts

```diff
@@ -0,0 +1,51 @@
+import { join } from "path";
```

Comment:
@AndreasArvidsson Ok I switched to exposing state via file.  Just copied some basic stuff from command server; should be fairly robust